monitoring/infrastructure.py

```diff
@@ -13,24 +13,25 @@
 # IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
 # CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
+from aws_cdk import CfnResource
+from aws_cdk import Fn
```

Comment:
I'd prefer to keep the module namespace for the reader to quicker understand which module they belong to:
```suggestion
import aws_cdk as cdk
```